crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -1235,21 +1203,23 @@ impl<P: PhaseItem, const I: usize> RenderCommand<P> for SetMeshViewBindGroup<I>
     }
 }
 
-pub struct SetMeshBindGroup<const I: usize>;
-impl<P: PhaseItem, const I: usize> RenderCommand<P> for SetMeshBindGroup<I> {
+pub struct SetMeshBindGroup<const I: usize, const PREPASS: bool>;
+impl<P: PhaseItem, const I: usize, const PREPASS: bool> RenderCommand<P>
+    for SetMeshBindGroup<I, PREPASS>
+{
     type Param = (
         SRes<MeshBindGroups>,
         SRes<RenderMeshInstances>,
         SRes<SkinIndices>,
         SRes<MorphIndices>,
     );
-    type ViewWorldQuery = ();
+    type ViewWorldQuery = Has<MotionVectorPrepass>;
     type ItemWorldQuery = ();
 
     #[inline]
     fn render<'w>(
         item: &P,
-        _view: (),
+        is_motion_vectors: bool,
```

Comment:
```suggestion
        has_motion_vectors: bool,
```

Reply:
Funnily, in the initial version of this PR, it was `has_motion_vectors`, and I changed it afterward. I put it back to `has_motion_vectors`